src/hooks/useCompanyTags.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, useContext } from 'react'
+import { getCompanyTags } from '../service/tagsByCompany'
+import { getTags } from '../service/tags'
+import Context from '../context/appContext'
+
+const useCompanyTags = () => {
+  const { selectedCompanyID } = useContext(Context).company
+  const [listOfTags, setListOfTags] = useState([])
+  const [tagsByCompany, setTagsByCompany] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+
+  useEffect(() => {
+    initTagsByCompany()
+    return () => setDefaultValues()
+  }, [])
+
+  const setDefaultValues = () => {
+    setIsLoading(false)
+  }
+
+  const initTagsByCompany = async () => {
+    const tagsByCompanyResponse = await getTagsByCompany()
+    setTagsByCompany(tagsByCompanyResponse)
+    const tagsListResponse = await getListOfTags()
+    setListOfTags(tagsListResponse)
+  }
+
+  const getListOfTags = async () => {
+    try {
+      setIsLoading(true)
+      const result = await getTags({})
+      const { tags } = destructuring(result)
```

Comment:
Autocomplete is not loading, I think It needs the isLoading to set to false

```suggestion
      const { tags } = destructuring(result)
      setIsLoading(false)
```